confgenerator/logging_processors.go

```diff
@@ -364,3 +368,66 @@ end`, lua))...)
 func init() {
 	LoggingProcessorTypes.RegisterType(func() LoggingProcessor { return &LoggingProcessorExcludeLogs{} })
 }
+
+// A LoggingProcessorGceMetadataAttributes annotates logs with labels
+// corresponding to instance attributes from the GCE metadata server.
+type LoggingProcessorGceMetadataAttributes struct {
+	ConfigComponent `yaml:",inline"`
+	Include         []string `yaml:"include" validate:"dive,gt=0,glob"`
+	Exclude         []string `yaml:"exclude" validate:"dive,gt=0,glob"`
+	Static          *bool `yaml:"static" validate:"required,eq=true"`
```

Comment:
You can just use `oneof` with a single enum value. No need to add a new validation type.